puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/packaging/pulpcore-packaging-rpm-pr-test.yaml

```diff
@@ -0,0 +1,20 @@
+---
+- job:
+    name: pulpcore-packaging-rpm-pr-test
+    project-type: pipeline
+    concurrent: true
+    sandbox: true
+    properties:
+      - github:
+          url: https://github.com/theforeman/pulpcore-packaging
+    triggers:
+      - github_pr_rpm:
+          context: 'rpm'
```

Comment:
wonder if this should be `pulpcore` instead?

Reply:
Since you have `rpm/x.y`, I think it's fine to keep it that way.

Reply:
Agreed, given the choice of `rpm/` pattern for branches, this follows on.